wiki/_README.md

```diff
@@ -0,0 +1,53 @@
+_Certified fresh 230808 by Graham Johnson._
+
+# Navigating the docs
+
+For a complete list of entries, see our [Table of Contents](_TOC.md)
+
+# Updating the docs: How & when
+
+Whenever a code change PR breaks (or makes obsolete) existing documentation, the relevant entry should be updated. This docs update should be included within the PR itself, as part of the PR—just as one would update unit tests alongside a breaking code change.
```

Comment:
I would like us to approach a standard for locating documentation as well as just when to update. My opinion is that, in the long term, docs should live alongside relevant code, rather than in a separate folder. However also would like to get some insight on what best practices are here. We may want to look at how Github internally does documentation and see what we can copy from their practices.

Reply:
10-4! I'll add this to my research to-do, and the ongoing philosophy-of-docs discussion ticket #4353.